buildSrc/src/main/kotlin/spotless-java-extension.gradle.kts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2025-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.diffplug.spotless.FormatterFunc
+import com.diffplug.spotless.FormatterStep
+import java.io.Serializable
+
+plugins {
+    id("com.diffplug.spotless")
+}
+
+spotless {
+    java {
+        addStep(
+            FormatterStep.create(
+                "multilineFormatter",
+                MultilineFormatter(),
+                { formatter -> FormatterFunc { input -> formatter.format(input) } }))
+    }
+}
+
+class MultilineFormatter : Serializable {
```

Comment:
I'm interested in the custom multiline formatting - what made you go with it?

Reply:
It is because we require the usage of multiline a lot in our project, especially for the following integration test pattern:
```
var expectedDocument = BsonDocument.parse(
            """
            {
                _id: 1,
                title: "War and Peace",
                author: "Leo Tolstoy",
                publishYear: 1867
            }""");
assertCollectionContainsExactly(expectedDocument);
```
previously the ugly alignment issue troubles coder and code reviewers a lot. Finally we create a Spotless step to make it hassle-free once for all.

I noticed the multiline alignment issue seems present in your branch as well (not in Java but in Kotlin), so this spotless customization might be useful to you as well, though not as necessary as in our usage.

Reply:
I like that - thanks for sharing.